src/main/resources/templates/home.html

```diff
@@ -10,6 +10,7 @@
 <main>
   <h1>Get Vaccinated</h1>
   <p><button th:onclick="showWarning()">Schedule a Vaccination</button></p>
+  <p><button onclick="redirect()">Report side effects</button></p>
```

Comment:
This could be more simply implemented as an `<a>` tag that links to the destination page.

Reply:
If you want to leave it as a button, to simplify having consistent styling, that's fine too. The extra closing `</script>` is the primary concern.